web/modules/custom/solution/solution.module

```diff
@@ -421,14 +421,7 @@ function solution_build_related_solutions_list(array $ids) {
     $element = $view_builder->view($referenced_solution, 'view_mode_tile', $referenced_solution->language()->getId());
     $elements[$delta] = [
       '#type' => 'container',
-      '#attributes' => [
-        'class' => [
-          'listing__item',
-          'listing__item--tile',
-          'mdl-cell',
-          'mdl-cell--4-col',
-        ],
-      ],
+      '#extra_suggestion' => 'container_grid_item',
```

Comment:
Hi @tom-waat adding reply to this note in the jira ticket

> Related solutions - this field is implemented in differently than other entity referenced fields, so I wasn't able to implement block styling there (grid is implemented). In fact in solution manage display there are two similar fields: "Related solutions: explicit + EIRA" and "Related Solutions". Only the first, which is not implemented correctly as entity referenced field, is displayed. I think that is needs to be reviewed.

Spoke with @sandervd, please feel free to modify anything in the `solution_build_related_solutions_list()` to re-use the helpers as in the other entity reference field instances.